src/azure-cli-core/azure/cli/core/auth/msal_authentication.py

```diff
@@ -65,8 +65,9 @@ def get_token(self, *scopes, claims=None, **kwargs):
 
         if claims:
             logger.warning('Acquiring new access token silently for tenant %s with claims challenge: %s',
-                           self.authority.tenant, claims)
-        result = self.acquire_token_silent_with_error(list(scopes), self._account, claims_challenge=claims, **kwargs)
+                           self._msal_app.authority.tenant, claims)
+        result = self._msal_app.acquire_token_silent_with_error(list(scopes), self._account, claims_challenge=claims,
```

Comment:
By saving `msal.PublicClientApplication` instance to `self._msal_app`, `self._account` is no longer saved as an attribute of `msal.PublicClientApplication`. This avoids the possible conflict of `msal.PublicClientApplication` introducing an `_account` attribute of its own.